modules/sdk-lib-mpc/src/tss/ecdsa-dkls/dkg.ts

```diff
@@ -1,22 +1,70 @@
 import { KeygenSession, Keyshare, Message } from '@silencelaboratories/dkls-wasm-ll-node';
-import { DeserializedBroadcastMessage, DeserializedMessages, DkgState } from './types';
+import { DeserializedBroadcastMessage, DeserializedMessages, DkgState, RetrofitData } from './types';
 import { decode } from 'cbor';
+import { encode } from 'cbor-x';
```

Comment:
do we need both cbor and cbor-x?

Reply:
good point. we actually need cbor-x since the encode from cbor does not follow expected behaviour and the encoded cbor is not what the dkls code is expecting. will see if i can change uses of cbor to cbor-x so i can remove cbor.

Reply:
done